src/VisualStudio/LiveShare/Impl/AbstractClassificationsHandler.cs

```diff
@@ -1,56 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Classification;
-using Microsoft.CodeAnalysis.LanguageServer;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.LanguageServices.LiveShare.CustomProtocol;
-using Microsoft.VisualStudio.LiveShare.LanguageServices;
-
-namespace Microsoft.VisualStudio.LanguageServices.LiveShare
-{
-    /// <summary>
-    /// Handler for a request to classify the document. This is used for semantic colorization and only works for C#\VB.
-    /// TODO - Move once defined as a custom protocol.
-    /// Note, this must return object instead of ClassificationSpan b/c liveshare uses dynamic to convert handler results.
-    /// Unfortunately, ClassificationSpan is an internal type and cannot be defined in the external access layer.
-    /// </summary>
-    internal abstract class AbstractClassificationsHandler : ILspRequestHandler<ClassificationParams, object[], Solution>
```

Comment:
All of the client side code for this should be deleted too
https://github.com/dotnet/roslyn/tree/master/src/VisualStudio/LiveShare/Impl/Client/Classification

Also the custom type - https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/LiveShare/Impl/CustomProtocol/Classifications.cs
and custom method name - https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/LiveShare/Impl/CustomProtocol/RoslynMethods.cs#L10